cmake/packaging/windows.cmake

```diff
@@ -39,9 +39,15 @@ install(DIRECTORY "${SUNSHINE_SOURCE_ASSETS_DIR}/windows/misc/gamepad/"
 install(DIRECTORY "${SUNSHINE_SOURCE_ASSETS_DIR}/windows/assets/"
         DESTINATION "${SUNSHINE_ASSETS_DIR}"
         COMPONENT assets)
-# copy assets to build directory, for running without install
+
+# copy assets (excluding shaders) to build directory, for running without install
 file(COPY "${SUNSHINE_SOURCE_ASSETS_DIR}/windows/assets/"
-        DESTINATION "${CMAKE_BINARY_DIR}/assets")
+        DESTINATION "${CMAKE_BINARY_DIR}/assets"
+        PATTERN "shaders" EXCLUDE)
+# use junction for shaders directory
+file(TO_NATIVE_PATH "${SUNSHINE_SOURCE_ASSETS_DIR}/windows/assets/shaders" shaders_in_build_src_native)
+file(TO_NATIVE_PATH "${CMAKE_BINARY_DIR}/assets/shaders" shaders_in_build_dest_native)
+execute_process(COMMAND cmd.exe /c mklink /J "${shaders_in_build_dest_native}" "${shaders_in_build_src_native}")
```

Comment:
I think we can use cmake to create a symbolic link instead. https://cmake.org/cmake/help/latest/command/file.html#create-link

Reply:
On Unix-like sure.
On Windows it's complicated. Symlinks require UAC elevation unless usermode symlinks are enabled in group policies or Windows is in Developer mode. Directory junctions always work without elevation.

Reply:
Do we need to go through cmd.exe or can we invoke `mklink` directly?

Reply:
Yes, `mklink` is a command of cmd shell, and there is no `mklink.exe`.